LondonBritishLibrary/orient/BLorient635.xml

```diff
@@ -236,9 +235,12 @@ schematypens="http://relaxng.org/ns/structure/1.0"
 
                     <history>
                         <origin>
-                            <origDate notBefore="1650" notAfter="1699" evidence="lettering"/>
+                            <origDate notBefore="1680" notAfter="1611" evidence="internal-date">Shortly before 1711.</origDate>
```

Comment:
can you explain where in the manuscript the date is given? I didn't find it, reading the file only quickly

Reply:
There are no dates, there is only the name of King Tewoflos, written in the same hand as the main text. He ruled 1708-1711, the manuscript was probably written shortly / some time before that.